### PR TITLE
Create Release.md

### PR DESCRIPTION
**Pull request by connyduck**
So everyone knows the things I do before a release and I could maybe hand this over to someone else.

### CODE REVIEW

**Review by nikclayton — COMMENTED** [review]
Just added some suggestions for future work. I'm not suggesting they need to be implemented before this PR is merged.

File: Release.md
Comment (charlag) [inline Release.md:15]: Can we please not use "sanity"?

File: Release.md
Comment (charlag) [inline Release.md:32]: Same

**Review by Tak — APPROVED** [review]

File: Release.md
Comment (connyduck) [reply Release.md:15]: yes, very good point

File: Release.md
Comment (connyduck) [reply Release.md:41]: I'd leave that to decide for the releasing person

File: Release.md
Comment (connyduck) [reply Release.md:14]: There are multiple ways to do it and it should be automated with bitrise anyway so I'll not bother with a detailed description for now.

**Pull request merged**
